src/main/java/org/ice4j/ice/Agent.java

```diff
@@ -1541,19 +1541,8 @@ protected void incomingCheckReceived(TransportAddress remoteAddress,
                                          String           localUFrag,
                                          boolean          useCandidate)
     {
-        if (isOver())
```

Comment:
I removed the comment, because I don't think it's accurate anymore. It could be a BR on the selected pair, or a PR for another succeeded pair, or a completely new pair.